synths/core-synths.scd

```diff
@@ -284,6 +284,18 @@ live coding them requires that you have your SuperDirt instance in an environmen
 
 	}, [\ir, \ir]).add;
 
+	// Tremolo audio DSP effect definition. Params: tremolorate (Hz); tremolodepth (amplitude)
+	SynthDef("tremolo" ++ numChannels, { |out, tremolorate = 1.0, tremolodepth = 0.5 |
+		var signal, tremolowave;
+
+		tremolodepth = tremolodepth * 0.5;
+		tremolowave = (1 - tremolodepth) + SinOsc.ar(tremolorate, 0, tremolodepth);
```

Comment:
(edited)

When the rate is very low, this will mute the sound, because the sine wave starts at `0`. So you could write: `SinOsc.ar(tremolorate, 0.5pi).range(0, tremolodepth)` to make it start at full level.

also, `tremolodepth = tremolodepth.min(1)`